app/src/test/java/com/example/mykotlincalculator21/mvp/presenter/CalculatorPresenterTest.kt

```diff
@@ -0,0 +1,324 @@
+package com.example.mykotlincalculator21.mvp.presenter
+
+import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
+import com.example.mykotlincalculator21.mvp.model.CalculatorModel
+import com.example.mykotlincalculator21.mvp.view.CalculatorView
+import com.example.mykotlincalculator21.utils.ErrorUtils
+import com.example.mykotlincalculator21.utils.StringUtils.EMPTY_STRING
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_DIVIDE
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_MINUS
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_MULTIPLY
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_SUM
+import com.example.mykotlincalculator21.utils.StringUtilsTest.NUMBER_FIVE
+import com.example.mykotlincalculator21.utils.StringUtilsTest.NUMBER_FOUR
+import com.example.mykotlincalculator21.utils.StringUtilsTest.NUMBER_THREE
+import com.example.mykotlincalculator21.utils.StringUtilsTest.NUMBER_TWO
+import com.example.mykotlincalculator21.utils.StringUtilsTest.NUMBER_ZERO
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class CalculatorPresenterTest {
+    private lateinit var presenter: CalculatorContract.CalculatorPresenterContract
+    private lateinit var model: CalculatorContract.CalculatorModelContract
+    private var view: CalculatorView = mock()
+
+    @Before
+    fun setup() {
+        model = CalculatorModel()
+        presenter = CalculatorPresenter(model, view)
+    }
+
+    @Test
+    fun eraseTest() {
+        presenter.erase()
```

Comment:
you should set the model with data, to test if it really delete the model. cause by default i assume is empty

Reply:
Done thx